ui/tests/integration/components/mfa-login-enforcement-header-test.js

```diff
@@ -50,12 +51,11 @@ module('Integration | Component | mfa-login-enforcement-header', function (hooks
     assert
       .dom('[data-test-mleh-description]')
       .includesText('An enforcement includes the authentication types', 'Description renders');
-
     for (const option of ['new', 'existing', 'skip']) {
       await click(`[data-test-mleh-radio="${option}"] input`);
       assert.equal(this.value, option, 'Value is updated on radio select');
       if (option === 'existing') {
-        await click('.ember-basic-dropdown-trigger');
+        await clickTrigger();
```

Comment:
nice cleanup

Reply:
the test was failing for some reason with the previous selector 🙃